lib/systemFlake.nix

```
@@ -94,8 +101,16 @@ let
   getChannels = system: self.pkgs.${system};
   getNixpkgs = host: (getChannels host.system).${host.channelName};
 
-  configurationBuilder = hostname: host': (
+  configurationBuilder = reverseDnsFqdn: host': (
```

In terms of naming, I thought `reverseDns` is generally the PTR record for IP to dns, but here it seems your using it in a more literal way as the dns string but flipped.

Can that be clarified with a comment or name change(flippedDns?)

---

```suggestion
  configurationBuilder = fqdn: host': (
  # fqdn => example: myhost.example.com
```

---

Wiki has it as: https://en.wikipedia.org/wiki/Reverse_domain_name_notation

---

Maybe reverseFqdn or reverseFqdnNotation
```suggestion
  configurationBuilder = reverseFqdn: host': (
  # example: com.example.myhost
```

---

You are right, addressed in https://github.com/gytis-ivaskevicius/flake-utils-plus/pull/69/commits/e834947e8b2bacee6fe2880fbfa557b5ef539ff8